lighthouse-core/audits/autocomplete.js

```diff
@@ -25,10 +26,165 @@ const UIStrings = {
    'effort, consider enabling autocomplete by setting the `autocomplete` ' +
    'attribute to a valid value.' +
   ' [Learn more](https://developers.google.com/web/fundamentals/design-and-ux/input/forms#use_metadata_to_enable_auto-complete)',
+  /** Label for a column in a data table; entries will be the autocomplete token suggestions based on heuristics. */
+  columnAutocompleteSuggestions: 'Autocomplete Suggested Token',
+  /** Label for a column in a data table; entries will be the incorrect autocomplete prefix tokens or prompting user to review them. */
+  columnAutocompleteCurrent: 'Autocomplete Current Value',
+  /**
+   * @description Warning that autocomplete token is invalid.
+   * @example {invalid-token name} token
+   * @example {<autocomplete="invalid-token name">} snippet
+   */
+  warningInvalid: 'Autocomplete token(s): "{token}" is invalid in {snippet}',
+  /**
+   * @description Warning that autocomplete token order is invalid.
+   * @example {mobile section-red cc-name} tokens
+   * @example {<autocomplete="mobile section-red cc-name">} snippet
+   */
+  warningOrder: 'Review order of tokens: "{tokens}" in {snippet}',
+  /* Entry for under the Autocomplete Suggested Token Column that tells users to review the ordering of their tokens if they are valid. */
```

Comment:
```suggestion
  /** Entry for under the Autocomplete Suggested Token Column that tells users to review the ordering of their tokens if they are valid. */
```

these need double star to work :) 

![image](https://user-images.githubusercontent.com/39191/91912411-ed956900-ec67-11ea-9850-5b8e684801f1.png)